.circleci/config.yml

```diff
@@ -169,11 +169,20 @@ jobs:
       - *restore_node_modules
       - run:
           name: Download artifacts for base revision
+          # FIXME: I intentionally broke this step to test that it recovers
+          # gracefully. Undo before landing.
           command: |
-              git fetch origin master
-              cd ./scripts/release && yarn && cd ../../
-              scripts/release/download-experimental-build.js --commit=$(git merge-base HEAD origin/master)
-              mv ./build2 ./base-build
+            false
+          # command: |
+          #     git fetch origin master
+          #     cd ./scripts/release && yarn && cd ../../
+          #     scripts/release/download-experimental-build.js --commit=$(git merge-base HEAD origin/master)
+          #     mv ./build2 ./base-build
+      - run:
+          name: Exit gracefully
+          command: |
+              echo "Failed to download build artifacts. This likely means the base revision is broken. Try rebasing onto the latest changes."
+          when: on_fail
```

Comment:
Welp this didn't work like I thought it would :D

Reply:
Ah I think maybe this will work: https://support.circleci.com/hc/en-us/articles/360015562253-Conditionally-end-a-running-job-gracefully